packages/bits/package.json

```diff
@@ -89,25 +89,17 @@
   },
   "analyze": false,
   "peerDependencies": {
-    "@angular-devkit/schematics": "^11.1.4",
     "@angular/animations": "^11.1.2",
     "@angular/cdk": "^11.1.1",
-    "@angular/common": "^11.1.2",
-    "@angular/compiler": "^11.1.2",
     "@angular/core": "^11.1.2",
     "@angular/forms": "^11.1.2",
     "@angular/localize": "^11.1.2",
     "@angular/platform-browser": "^11.1.2",
-    "@angular/platform-browser-dynamic": "^11.1.2",
     "@angular/router": "^11.1.2",
-    "@schematics/angular": "^11.1.4",
-    "core-js": "^2.6.9",
     "highlight.js": "^10.6.0",
     "lodash": "^4.17.21",
     "moment": "^2.25.3",
-    "moment-timezone": "^0.5.27",
-    "rxjs": "^6.5.3",
```

Comment:
rxjs and zone.js are required by angular/core, so not necessary to mention them here.